### PR TITLE
Add in the KendoReact Grid

### PR DESCRIPTION
**Pull request by bergenhem**
Hopefully I set up the PR correctly! Let me know if there are any items that I need to address!

### CODE REVIEW

File: source/projects/kendoreact-grid.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: KendoReact-Grid
+repo: telerik/kendo-react
+direct: https://github.com/telerik/kendo-react
+homepage: https://www.kendoreact.com
+examples: https://www.telerik.com/kendo-react-ui/components/grid/
+license: KendoReact License
+bower: n/a
+npm: @progress/kendo-react-grid
```
Comment (budnix) [inline source/projects/kendoreact-grid.md]:
```suggestion
npm: \@progress/kendo-react-grid
```
To make this markdown parsable  `@` sign have to be escaped :)

File: source/projects/kendoreact-grid.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* Virtualization
+* Column reordering
+* Column resizing
+* Detail templates
+* Globalization
+
+#### Preview:
+
+##### KendoReact Grid with Sorting, Paging, Filtering, and Grouping
+![KendoReact Grid with Sorting, Paging, Filtering, and Grouping](/images/libraries/kendoreact-grid/kendoreact-grid-features-enabled.png "KendoReact Grid with Sorting, Paging, Filtering, and Groupingr")
```
Comment (budnix) [inline source/projects/kendoreact-grid.md]:
```suggestion
![KendoReact Grid with Sorting, Paging, Filtering, and Grouping](/images/libraries/kendoreact-grid/kendoreact-grid-features-enabled.png "KendoReact Grid with Sorting, Paging, Filtering, and Grouping")
```
Fix typo

**Comment by bergenhem** [comment]
Thanks for reviewing things so quickly! I _believe_ I resolved everything through the GitHub interface since the edits were overall minor, but let me know if I missed anything :)

**Comment by budnix** [comment]
I've deployed the changes. The grid is available online at https://jspreadsheets.com/kendoreact-grid.html.

**Comment by bergenhem** [comment]
Awesome - thanks so much :)

**Pull request merged**